### PR TITLE
Implement shadow_offset

### PR DESCRIPTION
**Pull request by famfo**
I had to rebase the code so a new PR.

See: #225 
Fixes: #99

### CODE REVIEW

**Review by ErikReider — COMMENTED** [review]
LGTM! I'll wait for Will's approval before merging though

File: sway/sway.5.scd
Comment (WillPower3309) [inline sway/sway.5.scd:693]: Nit: wouldn't only apply to default shadows

File: sway/desktop/render.c
Comment (WillPower3309) [inline sway/desktop/render.c:353]: Q: wouldn't we want to add the offsets?

**Review by WillPower3309 — COMMENTED** [review]
One nit and one question! Otherwise LGTM!

Has this been tested on scaled and rotated outputs?

**Comment by ErikReider** [comment]
> Has this been tested on scaled and rotated outputs?

Tested and works :)

File: sway/desktop/render.c
Comment (famfo) [reply sway/desktop/render.c:353]: When adding the offsets, the box gets moved into the opposite direction.

Comment (WillPower3309) [reply sway/desktop/render.c:353]: Ah yes, missed the double negative of subtracting from a subtracted value

**Comment by WillPower3309** [comment]
Awesome, thanks for the work here! Just want to give it a manual test and then I'll go ahead and merge it

**Review by WillPower3309 — APPROVED** [review]

**Pull request merged**
